### PR TITLE
[Storage] Support new Storage Account Sku Standard_GZRS and Standard_RAGZRS

### PR DESCRIPTION
**Pull request by blueww**
## Description

Powershell design pass review in: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/249
Only add 2 new SkuName : Standard_GZRS and Standard_RAGZRS
Since this feature still not work on server, so not add Unit test.
Will add it later when server work. 

Also re-generate help file.

I have also upgrade the version to from 1.2.1-preview to 1.3.1-preview.
Since this is bases on the lastest master, it contains some not released Storage change in master branch, so also add these change to this release change log.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by blueww** [comment]
sign build: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign-esrp/353/

File: src/Storage/Storage.Management/Az.Storage.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.2.1'
+ModuleVersion = '1.3.1'
```
Comment (markcowl) [inline src/Storage/Storage.Management/Az.Storage.psd1]: If you are updating this, you should also update AssemblyInfo.cs

Comment (blueww) [reply src/Storage/Storage.Management/Az.Storage.psd1]: fixed， thanks!

**Comment by blueww** [comment]
@markcowl 
I have fixed you comment for version in AssemblyInfo.cs.
But I don't find `.travix.yml `in the root of the repro, so seems this is not a problem in this branch.

**Comment by markcowl** [comment]
@blueww  got it - that's why the build was waiting.  Updated branch configuration to fix.  Please let me know once the server tests have passed.

**Comment by blueww** [comment]
Thanks Mark for the review!
Will keep you updated. 
BTW, you are also in the thread for the deployment progress.

**Comment by MiYanni** [comment]
@blueww Any update on the server tests?

**Comment by blueww** [comment]
@MiYanni 
Thanks for the follow up!
The server is still not ready, so we can't test it.

**Comment by markcowl** [comment]
@blueww Closing this as it is not ready, please reopen when the service is testable

**Pull request closed without merging**